src/compiler/scala/tools/nsc/transform/CleanUp.scala

```diff
@@ -524,4 +597,24 @@ abstract class CleanUp extends Statics with Transform with ast.TreeDSL {
 
   } // CleanUpTransformer
 
+
+  val BOX = TermName("box")
+  val objectMethods = Map[Name, TermName](
+    termNames.hashCode_ -> termNames.hashCode_,
+    termNames.toString_ -> termNames.toString_
+    )
+  val doubleAndFloatRedirectMethods = Map[Name, TermName](
+    TermName("isNaN") -> TermName("isNaN"),
+    TermName("isInfinite") -> TermName("isInfinite")
+    )
+  val javaNumberConversions         = Map[Name, TermName](
+    TermName("byteValue") -> termNames.toByte,
+    TermName("shortValue") -> termNames.toShort,
+    TermName("intValue") -> termNames.toInt,
+    TermName("longValue") -> termNames.toLong,
+    TermName("floatValue") -> termNames.toFloat,
+    TermName("doubleValue") -> termNames.toDouble
```

Comment:
The new ones too, IMO.  I think the idea is that it's best to have these all defined once and be in a known location.